cmake_modules/commonSetup.cmake

```diff
@@ -885,6 +885,9 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
       if(USE_TBBMALLOC)
           message(STATUS "Enabled use of TBBMALLOC")
           add_definitions (-D_USE_TBBMALLOC)
+          if(USE_TBBMALLOC_ROXIE)
+              message(STATUS "Enabled use of TBBMALLOC_ROXIE")
+          endif(USE_TBBMALLOC_ROXIE)
```

Comment:
Added to be consistent with TBBMALLOC_ROXIE cmake setting so that build log would show if enabled.